activiti-cloud-connectors-starter/src/main/java/org/activiti/cloud/connectors/starter/model/IntegrationRequestEvent.java

```diff
@@ -33,6 +38,18 @@
     public IntegrationRequestEvent() {
     }
 
+    public IntegrationRequestEvent(String id,
```

Comment:
This constructor seems never be used. The json deserialisation  will use the default constructor.